tags/tag-developer-experience/charter.md

```diff
@@ -1 +1,67 @@
-Charter content here
\ No newline at end of file
+# Mission
+
+The mission of the CNCF TAG Developer Experience (TAG: DevEx) is to improve the end-to-end experience for developers building, testing, deploying, and maintaining cloud native applications. We emphasize outcomes over dogma, focusing on unifying developers, platforms to drive modern software excellence, system resilience, and business value. We aim to provide guidance, foster collaboration, and support CNCF projects that enhance productivity, usability, user experience, and improve developer satisfaction within the cloud native ecosystem.
+
+# Scope
+
+TAG: Developer Experience (DevEx) focuses on enhancing the end-to-end experience of developers building cloud native applications. Our scope encompasses key concerns across various areas, including microservices, databases, streaming and messaging systems, AI, API management, and developer programming languages and frameworks, in addition to traditional developer experience.
+We explore and guide on:
+
+* Cloud Native application excellence
+* Inner/outer loop best practices
+* Observability and debugging from a developer’s perspective
+* Usability, DevEX research, and DevEX metrics across CNCF projects
+* AI development-related Developer Experience 
+
+Our work involves identifying gaps, promoting best practices, and supporting CNCF projects and communities to enhance the productivity and satisfaction of developers working with cloud native technologies. We aim to bridge tools, platforms, and practices to create a cohesive and frictionless developer experience across the cloud native ecosystem.
+
+More specifically, we:
+
+* Provide guidance and thought leadership on improving developer workflows and experience in cloud native application development
+* Promote secure-by-default developer workflows by integrating security and compliance considerations into the developer experience
+* Collaborate with CNCF projects to surface and resolve developer experience challenges
+* Conduct studies, gathering feedback from end users and contributors, and providing recommendations
+* Facilitate interoperability and standardization across developer tools in the CNCF ecosystem
+* Act as a bridge to empower developers and project maintainers
```

Comment:
Seems unfinished. What is the change you'd like to bring to developers and project maintainers?